mk/spksrc.generate-digests.mk

```diff
@@ -3,8 +3,10 @@
 # include this file after the rule "all:"
 #
 
+ifeq ($(strip $(PKG_DIST_ARCHS)),)
+
 $(DIGESTS_FILE): download
-	@$(MSG) "Generating digests for $(NAME)"
+	@$(MSG) "Generate digests for $(NAME) - PKG_DIST_ARCH=$(PKG_DIST_ARCH)"
```

Comment:
if `$(PKG_DIST_ARCHS)` is empty, calling `$(MSG)` with it will also be empty?
Ah, now I see, really not obvious of using `PKG_DIST_ARCH` with and without an `S`.
To differentiate them, perhaps something like `PKG_DIST_ARCH_LIST` instead.
(while I get of keeping things aligned with CPU ARCHS for instance)

Reply:
Yes we already have `*_ARCH` and `*_ARCHS` in `spksrc.archs.mk` and I don't like to use a diffeent approach here.

Reply:
Agreed, and makes me wonder if we shouldn't revisit that again sooner or later to use a `_LIST` naming.... 
In this specific case it really is easy to mix things up and would benefit of getting additional clarity.

Reply:
BTW I alredy added the new Makefile variables in the [wiki](https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables)

Reply:
it's not a problem, it's just a little misleading.